babel.config.js

```diff
@@ -43,7 +44,7 @@ const config = {
 	] ),
 	env: {
 		test: {
-			presets: [ [ '@babel/env', { targets: { node: 'current' } } ] ],
+			presets: [ [ '@babel/env', { targets: { node: 'current' } } ], '@babel/preset-typescript' ],
```

Comment:
This one is not needed. In the `env` config, we only specify additional presets that are not present in the base config.